app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,6 @@
+class MoviesController < ApplicationController
+  def show
+    @movie = Movie.find(params[:id])
+    @actors_average_age = @movie.average_age
```

Comment:
Just for the future -- this second instance var is not strictly necessary because you can call `@movie.average_age` in the view, too! Calling a model method from the view is just fine, we just don't want to call AR methods from the view